packages/expo-random/expo-module.config.json

```
@@ -1,6 +1,10 @@
 {
-  "platforms": ["ios"],
+  "name": "expo-random",
+  "platforms": ["ios", "android"],
   "ios": {
     "modules": ["RandomModule"]
+  },
+  "android": {
+    "modulesClassNames": ["expo.modules.random.RandomModule"]
```

Since the `ios` part already uses the new name (`modules`), it probably makes sense to use it here as well
```suggestion
    "modules": ["expo.modules.random.RandomModule"]
```